opentelemetry-api/tests/context/base_context.py

```diff
@@ -64,7 +64,7 @@ def test_attach(self):
             self.assertEqual("yyy", context.get_value("a"))
 
             with self.assertLogs(level=ERROR):
-                context.detach("some garbage")
+                context.detach("some garbage")  # type:ignore
```

Comment:
Why is the `type: ignore` needed?

```suggestion
                context.detach("some garbage")  # type: ignore[?]
```

Reply:
True thanks this was actually not necessary, slipped past me. I removed the `type:ignore`